packages/botid/package.json

```diff
@@ -0,0 +1,52 @@
+{
+  "name": "@better-auth/botid",
+  "author": "Bereket Engida",
+  "version": "1.0.0",
+  "main": "dist/index.cjs",
+  "license": "MIT",
+  "keywords": [
+    "vercel",
+    "auth",
+    "bot",
+    "bot-detection",
+    "is-human",
+    "botid",
+    "better-auth"
+  ],
+  "module": "dist/index.mjs",
+  "description": "Vercel BotID for Better Auth",
+  "scripts": {
+    "test": "vitest",
+    "build": "unbuild",
+    "typecheck": "tsc --noEmit",
+    "dev": "unbuild --watch"
+  },
+  "exports": {
+    ".": {
+      "types": "./dist/index.d.ts",
+      "import": "./dist/index.mjs",
+      "require": "./dist/index.cjs"
+    }
+  },
+  "typesVersions": {
+    "*": {
+      "*": [
+        "./dist/index.d.ts"
+      ],
+      "client": [
+        "./dist/client.d.ts"
+      ]
+    }
+  },
+  "dependencies": {
+    "better-auth": "workspace:^",
+    "zod": "^3.24.1"
+  },
+  "devDependencies": {
```

Comment:
use peerDeps

Refer to two commits: https://github.com/better-auth/better-auth/commit/d4343005fee136e16c6e1ba527806fc86a53daa2 https://github.com/better-auth/better-auth/commit/091aef9f2ebaec896943988d9aef35dc1c85ebf4

Reply:
https://github.com/better-auth/better-auth/pull/4295